src/NHibernate.Test/NHSpecificTest/NH3772/Fixture.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Linq;
+using NHibernate.Cfg.MappingSchema;
+using NHibernate.Mapping.ByCode;
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.NH3772
+{
+	public class Fixture : TestCaseMappingByCode
```

Comment:
Lacking `TestFixture` attribute, causing it to not generate an async counterpart.

Reply:
True

Reply:
It does not generate async code because `action` delegate declared outside of `Assert.That(...)`

Reply:
Ok, but it still should have the attribute for consistency with other tests.

Reply:
I've added the attribute